acceptance/localcluster/localcluster.go

```diff
@@ -242,7 +242,9 @@ func (l *Cluster) initCluster() {
 	maybePanic(c.Wait())
 }
 
-func (l *Cluster) startNode(i int) *Container {
+// StartNode creates and starts the container corresponding
+// to node 'i'. The container is saved in l.Nodes.
+func (l *Cluster) StartNode(i int) {
```

Comment:
There was when I was killing/starting nodes (eg: once killed, we have to create a new container).
I'll un-export this now that I'm using restart.